lightning/src/routing/network_graph.rs

```diff
@@ -208,8 +211,8 @@ where C::Target: chain::Access, L::Target: Logger
 			},
 			NetworkUpdate::NodeFailure { ref node_id, is_permanent } => {
 				let action = if is_permanent { "Removing" } else { "Disabling" };
-				log_debug!(self.logger, "{} node graph entry for {} due to a payment failure.", action, node_id);
-				self.network_graph.fail_node(node_id, is_permanent);
+				log_debug!(self.logger, "{} node graph entry for {:?} due to a payment failure.", action, node_id);
```

Comment:
I think instead of `{:?}` which prints a ton of crap, we should use `log_bytes!()` around the `node_id` parameter to just get the hex.

Reply:
Thanks for the catch! I remember looking at `log_bytes!()` earlier, but forgot about using it here.